storage-provider/server/src/config.rs

```diff
@@ -112,27 +112,27 @@ pub struct ConfigurationArgs {
 
     /// P2P Node type, can be either a bootstrap node or a registration node.
     #[serde(default = "NodeType::default")]
-    #[arg(long, default_value_t = NodeType::Bootstrap)]
+    #[arg(long, default_value_t = NodeType::Bootstrap, required = false)]
```

Comment:
Are we validating that later, that if no configuration file is passed, it'll require p2p_key and others anyways?